src/program/spoon/README.md

```diff
@@ -0,0 +1,41 @@
+## Inter-process sample program
+
+The `program/spoon` directory contains a sample (and test, and benchmark)
+for the inter_proc app.
```

Comment:
yes, it's a bad joke (not a fork and piles nicely, matching convex with concave).  i plead the [second hard thing](https://twitter.com/codinghorror/status/506010907021828096).